packages/category-feed-persoo/tests/Unit/PersooCategoryFeedItemTest.php

```diff
@@ -0,0 +1,127 @@
+
+    private FriendlyUrlFacade|MockObject $friendlyUrlFacadeMock;
+
+    private ImageFacade|MockObject $imageFacadeMock;
+
+    protected function setUp(): void
+    {
+        $this->friendlyUrlFacadeMock = $this->createMock(FriendlyUrlFacade::class);
+        $this->imageFacadeMock = $this->createMock(ImageFacade::class);
+        $categoryRepositoryMock = $this->createMock(CategoryRepository::class);
+
+        $this->persooFeedItemFactory = new PersooCategoryFeedItemFactory(
+            $this->friendlyUrlFacadeMock,
+            $this->imageFacadeMock,
+            $categoryRepositoryMock,
+        );
+
+        $this->defaultDomain = $this->createDomainConfigMock(
+            Domain::FIRST_DOMAIN_ID,
+            'https://example.com',
+            'en',
+        );
+
+        $this->defaultCategory = $this->createMock(Category::class);
+        $this->defaultCategory->method('getId')->willReturn(1);
+        $this->defaultCategory->method('getName')->with('en')->willReturn('category name');
+
+        $this->mockCategoryUrl($this->defaultCategory, $this->defaultDomain, 'https://example.com/category-1');
+    }
+
+    /**
+     * @param int $id
+     * @param string $url
+     * @param string $locale
+     * @return \Shopsys\FrameworkBundle\Component\Domain\Config\DomainConfig
+     */
+    private function createDomainConfigMock(int $id, string $url, string $locale): DomainConfig
+    {
+        $domainConfigMock = $this->createMock(DomainConfig::class);
+
+        $domainConfigMock->method('getId')->willReturn($id);
+        $domainConfigMock->method('getUrl')->willReturn($url);
+        $domainConfigMock->method('getLocale')->willReturn($locale);
+
+        return $domainConfigMock;
+    }
+
+    /**
+     * @param \Shopsys\FrameworkBundle\Model\Category\Category $category
+     * @param \Shopsys\FrameworkBundle\Component\Domain\Config\DomainConfig $domain
+     * @param string $url
+     */
+    private function mockCategoryUrl(Category $category, DomainConfig $domain, string $url): void
+    {
+        $this->friendlyUrlFacadeMock->method('getAbsoluteUrlByRouteNameAndEntityId')
+            ->with($domain->getId(), 'front_product_list', $category->getId())->willReturn($url);
+    }
+
+    /**
+     * @param \Shopsys\FrameworkBundle\Model\Category\Category $category
+     * @param \Shopsys\FrameworkBundle\Component\Domain\Config\DomainConfig $domain
+     * @param string $url
+     */
+    private function mockCategoryImageUrl(Category $category, DomainConfig $domain, string $url): void
+    {
+        $this->imageFacadeMock->method('getImageUrl')
+            ->with($domain, $category)->willReturn($url);
+    }
+
+    public function testMinimalPersooCategoryFeedItemIsCreatable(): void
+    {
+        $persooCategoryFeedItem = $this->persooFeedItemFactory->create($this->defaultCategory, $this->defaultDomain);
+
+        self::assertInstanceOf(PersooCategoryFeedItem::class, $persooCategoryFeedItem);
```

Comment:
![image](https://github.com/shopsys/shopsys/assets/10401898/9fe24f4e-e56c-46f9-9cd5-41e4e97221a0)

The inspection is right - the `create` method has a return type hint so `$persooCategoryFeedItem` will be an instance of `PersooCategoryFeedItem` without any doubt :slightly_smiling_face:

Reply:
I have removed it